Atlas/android/app/src/main/java/com/bounswe2017/group10/atlas/adapter/CommentRow.java

```diff
@@ -34,14 +45,34 @@ public void setText(String text) {
         this.text = text;
     }
 
+    public String getAvatar() {
+        return avatar;
+    }
+
+    public void setAvatar(String avatar) {
+        this.avatar = avatar;
+    }
+
     public CommentRow() {
 
     }
 
-    public CommentRow(String name, String date, String text) {
+    public CommentRow(String name, String date, String text, String avatar) {
         this.name = name;
+
+        String[] parts = date.split("[.]");
```

Comment:
I believe the functionality of this code piece is the same as ```setDate```. Can you call the function to reduce code duplication

Reply:
done.